src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -321,6 +321,7 @@ public MimeMessage sendRegistrationInviteToInstructor(String courseId, Instructo
         
     }
     
+    @SuppressWarnings("deprecation")
     public String sendJoinLinkToNewInstructor(InstructorAttributes instructor, String shortName, String institute) 
```

Comment:
Ok, I'll merge in these changes first later but we keep the discussion going as to whether it should not be deprecated and if it shouldn't, I open another issue to undeprecate it since it's not really related to what this PR's issue is about.